dev-tools/scripts/releaseWizard.yaml

```diff
@@ -1059,7 +1056,7 @@ groups:
         logfile: checkout-release-tag.log
         tee: true
       - !Command
-        cmd: ant documentation -Dversion={{ release_version }}
+        cmd: "{{ gradle_cmd }} documentation -Dversion={{ release_version }}"
```

Comment:
This must be `-Dversion.release=%s`, as gradle has some reserved meaning of `version`.